docs/configuring-playbook-appservice-draupnir-for-all.md

```diff
@@ -16,36 +16,65 @@ Normal Draupnir does come with the benefit of access to Synapse Admin features.
 
 Draupnir for all does not support external tooling like [MRU](https://mru.rory.gay) as it can't access Draupnir's user account.
 
-## Installation
+## Prerequisites
 
-### Create a main management room.
+### Create a main management room
 
-The playbook does not create a management room for your Main Draupnir. This task you have to do on your own.
+The playbook does not create a management room for your Main Draupnir. You **need to create the room manually** before setting up the bot.
+
+Note that the room must be unencrypted.
+
+<!-- TODO: enable Pantalaimon as configuring-playbook-bot-draupnir.md -->
 
 The management room has to be given an alias and be public when you are setting up the bot for the first time as the bot does not differentiate between invites and invites to the management room.
```

Comment:
I'm not really sure if it is fine to call this as the bot, since Draupnir for All is Appservice mode and different from the bot mode :thinking:

Reply:
The way I see things: the way the "bot" operates (appservice or not) is an implementation detail. As far as users are concerned, you're still talking to a Draupnir bot.